wapitiCore/attack/mod_upload.py

```diff
@@ -163,22 +162,21 @@ async def attack(self, request: Request, response: Optional[Response] = None):
                             self.network_errors += 1
                         else:
                             if payload_info.payload in link_response.content:
-                                await self.add_vuln_critical(
+                                await self.add_critical(
                                     request_id=request.path_id,
-                                    category=NAME,
+                                    finding_class=UnrestrictedUploadFinding,
                                     request=mutated_request,
                                     info=(
                                         "Unrestricted file upload vulnerability in the parameter "
                                         f"{parameter.display_name}"
                                     ),
                                     parameter=parameter.display_name,
-                                    wstg=WSTG_CODE
                                 )
 
                                 log_red("---")
                                 log_red(
                                     Messages.MSG_PARAM_INJECT,
-                                    NAME,
+                                    str(UnrestrictedUploadFinding),
```

Comment:
idem ?

Reply:
No, having a single point of thruth ensure no typo is introduced.